server/master/src/main/java/org/apache/accumulo/master/metrics/Metrics2ReplicationMetrics.java

```diff
@@ -58,6 +59,7 @@ protected void snapshot() {
     registry.add(PENDING_FILES, getNumFilesPendingReplication());
     registry.add(NUM_PEERS, getNumConfiguredPeers());
     registry.add(MAX_REPLICATION_THREADS, getMaxReplicationThreads());
+    registry.add(LATENCY, getLatencyInSeconds());
```

Comment:
Yes, not saying that having the maximum isn't helpful, but it paints a very skewed picture.

We have the primitives to collect and display metrics which paint a complete picture. I really think that we should use them instead of only displaying the maximum.